src/demo/share/jfc/SwingSet2/TableDemo.java

```diff
@@ -549,7 +549,13 @@ public JScrollPane createTable() {
             public int getRowCount() { return data.length;}
             public Object getValueAt(int row, int col) {return data[row][col];}
             public String getColumnName(int column) {return names[column];}
-            public Class<?> getColumnClass(int c) {return getValueAt(0, c).getClass();}
+            public Class<?> getColumnClass(int c) {
+                Object obj = getValueAt(0, c);
+                if (obj != null) {
+                    return obj.getClass();
+                }
+                return Object.class;
```

Comment:
Shall the ternary conditional operator be used here?
```suggestion
                return obj != null ? obj.getClass() : Object.class;
```